slides/03_workflow.qmd

```diff
@@ -1,10 +1,591 @@
 ---
-title: "A Package Engineering Workflow"
+title: "An R Package Engineering Workflow"
+subtitle: "BBS Course: Good Software Engineering Practice with R"
 author: "Friedrich"
+affiliation: "RPACT"
+institute: "RPACT"
```

Comment:
I would take this out - we also don't have this on the other slides ? folks have affiliations on the announcement

Reply:
You are right - is removed.